cypress.config.ts

```diff
@@ -0,0 +1,48 @@
+// eslint-disable-next-line node/no-unpublished-import
```

Comment:
Wait, `.ts` file in the frontend? Is that a cypress requirement?

Reply:
We can use either a `.ts` file or a `.js` file; https://docs.cypress.io/guides/references/migration-guide#Configuration-File-Changes. I didn't think much about it initially, but now that I think maybe for consistency (since we don't have any `.ts` files) it's better to have `.js` file there. I've went ahead and changed it.

Reply:
Ok, I was just surprised. It's better to have consistency with `js` but I wasn't requiring a change, I was trying to understand :slightly_smiling_face: